src/components/QuestionsGrid/QuestionsGrid.svelte

```diff
@@ -1,17 +1,22 @@
 <script lang="ts">
     import { QuestionCard } from '../../components'
     import { link } from 'svelte-spa-router'
+import { mount_component } from 'svelte/internal'
```

Comment:
What is this import doing? it doesn't seem to be used anywhere

Reply:
not sure why that's there...